gulpfile.js

```diff
@@ -269,14 +270,34 @@ function resolveInstallerExePath(artifactType) {
   return path.join(zaRoot, artifactName + '-' + pjson.version + artifactType + '-installer.exe');
 }
 
-// download all the installer dependencies so we can package them up into the .exe
+function copyFromPrefetchToDist(currentFile, destFile) {

```

Comment:
I see no reason why we need to copy all downloaded bits. We create 7z archive like this `7za.exe a installer.7z installer-dir/*`, can't we just add second path to this command? I would be both faster and would save 1GB on hard disk

`7za.exe a installer.7z installer-dir/* prefetch-dependencies/*`